Disable this test; its code was inadvertently omitted from the 2.0.3 distfile.
https://github.com/PortMidi/portmidi/issues/21
https://github.com/PortMidi/portmidi/commit/753dd2417b29e11e084ed4e03ca1cf359c6e07b5
--- pm_test/CMakeLists.txt.orig	2022-02-21 19:54:37.000000000 -0600
+++ pm_test/CMakeLists.txt	2022-05-23 00:28:40.000000000 -0500
@@ -38,5 +38,4 @@
 add_test(sendvirtual)
 add_test(multivirtual)
 add_test(virttest)
-add_test(pmlist)
 endif(WIN32)
